### PR TITLE
feat: add ability to set saml request signature

### PR DESCRIPTION
**Pull request by theburyat**

### CODE REVIEW

File: controllers/cert.go [hunk 1]
```diff
@@ -106,6 +106,27 @@ func (c *ApiController) GetGlobleCerts() {
...
 
+// GetCertsByScope
+// @Title GetCertsByScope
+// @Tag Cert API
+// @Description get certs by scope
+// @Param   owner     query    string  true        "The owner of the cert"
+// @Param   scope     query    string  true        "The scope of the cert"
+// @Success 200 {object} []object.Cert The Response object
+// @router /get-scope-certs [get]
+func (c *ApiController) GetCertsByScope() {
```
Comment (dplynskiy) [inline controllers/cert.go:117]: we can use getCerts with field = 'scope' and value instead creating new method?

File: controllers/cert.go [hunk 2]
```diff
@@ -39,7 +39,15 @@ func (c *ApiController) GetCerts() {
 	sortOrder := c.Input().Get("sortOrder")
 
 	if limit == "" || page == "" {
-		maskedCerts, err := object.GetMaskedCerts(object.GetCerts(owner))
+		var (
+			maskedCerts []*object.Cert
+			err         error
+		)
+		if field == "scope" && value != "" {
+			maskedCerts, err = object.GetMaskedCerts(object.GetCertsByScope(owner, value))
```
Comment (dplynskiy) [inline controllers/cert.go:47]: and we can use GetPaginationCerts instead creating GetCertsByScope

File: controllers/cert.go (hunk 1 above)
Comment (theburyat) [reply controllers/cert.go]: fixed

File: controllers/cert.go (hunk 2 above)
Comment (theburyat) [reply controllers/cert.go]: fixed

**Review by dplynskiy — APPROVED** [review]

**Pull request merged**